Tools/Scripts/webkitpy/browserperfdash/browserperfdash_runner.py

```diff
@@ -103,6 +103,15 @@ def _get_test_data_json_string(self, temp_result_file):
             del temp_result_json['debugOutput']
         return json.dumps(temp_result_json)
 
+    # urllib.request.urlopen always rises an exception when the http return code is not 200
```

Comment:
Typo: `s/rises/raises/`

```suggestion
    # urllib.request.urlopen always raises an exception when the http return code is not 200
```